README.md

```diff
@@ -299,27 +299,47 @@ $tnt->replace("test", array(1, 3, "smth completely different"));
 ### Tarantool::call
 
 ``` php
-public array Tarantool::call(string $procedure [, mixed args])
+public array Tarantool::call(string $procedure [, mixed args [, array $opts]])
 ```
 
 _**Description**_: Call stored procedure
 
 _**Parameters**_
 * `procedure`: String, procedure to call (mandatory)
-* `args`: Any value to pass to procdure as arguments (empty by default)
+* `args`: Any value to pass to procedure as arguments (empty by default)
+* `opts`: Array, options
+
+_**Options**_
+* call_17<br />
```

Comment:
Should not we stick with net.box terms and introduce the option `call_16` (which enabled the old behaviour), not `call_17`?

It is quite minor, since I don't find any common approach between connectors. However I don't find the answer for myself, so if you have thoughts how to do such things better, please, share.

Reply:
My thoughts. The default is call_16 mode, so if I want to change that it would be nice to enable something (true) rather than disable (false).

Reply:
I will quote Petrenko :"Still, it's up to you which solution you find a better one, and I'll implement it."

Reply:
We can think about it from another side: if we'll continue development of the connector, then we'll change default at some moment and the same reason will suggest to choose `call_16` name.

Nope, I don't know what is better, just want to understand arguments for `call_16` and for `call_17`.

Reply:
I'm not sure is it needed to change the default, just add the option to the constructor. Up to you.

Reply:
Since it is not clear what way is better, I would stick to the net.box way: `call_16`. tarantool-python also name the constructor option as `call_16`.

Reply:
Updated to call_16.